core/base/document.php

```diff
@@ -822,10 +822,8 @@ public function print_elements_with_wrapper( $elements_data = null ) {
 		}
 		?>
 		<div <?php echo Utils::render_html_attributes( $this->get_container_attributes() ); ?>>
-			<div class="elementor-inner">
-				<div class="elementor-section-wrap">
-					<?php $this->print_elements( $elements_data ); ?>
-				</div>
+			<div class="elementor-section-wrap">
```

Comment:
It's a class in use, see: https://github.com/elementor/elementor/blob/master/assets/dev/js/editor/views/preview.js#L10